src/index.js

```diff
@@ -1,3 +1,7 @@
 import auditor from "./auditor.js";
 
-auditor();
+window.AccessLint = {
+  audit({ url = "/accesslint/errors", raiseError = false }) {
```

Comment:
Expected ',' and instead saw '='.<br>Expected an identifier and instead saw '/accesslint/errors'.<br>Expected an identifier and instead saw 'false' (a reserved word).<br>'false' is defined but never used.